samples/wifi/provisioning/implementation_details.rst

```diff
@@ -0,0 +1,313 @@
+.. _wifi_provisioning_details:
+
+Implementation Details
```

Comment:
I have some questions to ask in the group chat and will provide more detail there, but wanted to summarize the general feedback or recurring items here rather than commenting on every instance:

* Headings and titles should use sentence case (Task and event handling), rather than Title Case With Many Capital Letters.
* The name of the sample including Service is confusing, as services have a specific meaning in places like nRF Cloud.
* SDK RST files have one sentence per line with no trailing space.
* Shall + verb should be just the third-person present tense of the verb, for example, shall respond -> responds.
* Shall be -> is.
* Present tense rather than future tense, except for some specific constructions that include a conditional (if X, the device will Y)
* Watch for subject-verb agreement errors where a singular subject has a plural verb, or the other way around. Details is provided -> details are provided, though this example is problematic for other reasons (passive voice) :)
* Check the [Tech Docs style guide](https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Terms%2C+abbreviations%2C+and+spelling+rules) for acceptable abbreviations. Bluetooth (registered trademark symbol) Low Energy (LE) for the first occurrence on a page, then Bluetooth LE (not BLE) after.
* Use the expanded phrase before using the abbreviation. We use AP before we specify *access point* almost at the end of the page.
* Message types shouldn't have capitalized names (request, response, info)
* Don't use Latin abbreviations like etc. and e.g.: see the style guide linked above for replacements.
* Commands should be in `backticks` (or properly formatted according to RST), as well as field and parameter names.
* Despite my using it in this feedback, we don't use 'should' or 'supposed'. Assume it works: The device takes action according to the command.
* Capitalization in table items. I will provide specific examples tomorrow.
* Some missing a/the articles. I will comment with specific examples tomorrow, as these are harder to spot.
* The 'Procedures' would be more accurately 'Operations' and not be their own headings. I have a list-based suggestion for this that I'll comment (it's in Markdown, sorry) in the relevant place. This also summarizes common response messages so we don't repeat them under each operation.
* Passive voice ('is <verb>ed') rather than active ('<verb>s') in a lot of places where it's not clear what is doing the action.
* Some information that we can just leave out or condense. I'll provide more on this tomorrow and would be happy to answer anything in chat.

I have line-by-line feedback in a file that I will turn into comments tomorrow, but wanted to provide feedback on the changes that need to happen across the page.

Reply:
I know this list looks overwhelming, but most of these are small changes that become second nature :)